lib/salus/report.rb

```diff
@@ -157,19 +158,27 @@ def to_s(verbose: false, wrap: WRAP, use_colors: false)
       output
     end
 
-    def to_yaml
-      YAML.dump(to_h)
+    def to_yaml(sort = false)
+      sort ? YAML.dump(to_h.deep_sort) : YAML.dump(to_h)
```

Comment:
Minor nit: You could make the code cleaner (only call dump once) with 
```ruby
res = sort ?  to_h.deep_sort, to_h
YAML.dump(res)
```

Similar for the other methods.